athenahealth/ratelimiter/redis.go

```diff
@@ -0,0 +1,71 @@
+package ratelimiter
+
+import (
+	"context"
+	"time"
+
+	"github.com/go-redis/redis/v8"
+	"github.com/go-redis/redis_rate/v9"
+)
+
+const redisKeyPreview = "athena_rate_limit:preview"
+const redisKeyProd = "athena_rate_limit:prod"
+
+const defaultRatePreivew = 5
```

Comment:
Can we put a comment or change these const names so it's easier to tell that these values are "per second"?